app/assets/stylesheets/workspace.scss

```diff
@@ -1,6 +1,14 @@
+.workspaces-show {
+  .container-fluid {
+    margin: 0;
+    padding: 0;
+  }
+}
+
 #viewer {
+  clear: both;
   display: block;
-  height: 600px;
-  position: relative;
+  height: calc(100% - 55px); // should be $navbar-height, but variable is not available
```

Comment:
This *may* be your problem: we're using the bootstrap gem, and according to it's docs (https://github.com/twbs/bootstrap-rubygem) these are the available variables - https://github.com/twbs/bootstrap-rubygem/blob/master/templates/project/_bootstrap-variables.scss#L582. None of the navbar height variables are available through the gem that are otherwise used in standard bootstrap (as in the variable list that @mejackreed posted above).

I could be wrong about this, but the docs in the Rails section include this line: "The available variables can be found [here](https://github.com/twbs/bootstrap-rubygem/blob/master/templates/project/_bootstrap-variables.scss."